docs/detections/rules-cross-cluster-search.asciidoc

```diff
@@ -0,0 +1,65 @@
+
+[discrete]
+[[set-up-ccs-rules]]
+=== Set up cross-cluster search in detection rules
+
+This section explains the general process for setting up cross-cluster search in detection rules. For specific instructions on each part of the process, refer to the linked documentation.
+
+. On the local cluster, {ref}/remote-clusters-connect.html[set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
+
+. On both the local and remote clusters, {ref}/remote-clusters-privileges.html#clusters-privileges-ccs-kibana[create a role for cross-cluster search privileges], and make sure the two roles have _identical_ names. Assign each role the following privileges:
+
+.. *Local cluster role*: Assign the `read` privilege to the indices you want to search, using _both_ the local and remote index patterns for each index. To specify a remote index, use the pattern `<remote_cluster_name>:<index_name>`.
++
+For example, if the remote cluster's name is `remote-security-data` and you want to query the `logs-*` indices, include both `logs-*` and `remote-security-data:logs-*` and assign them the `read` privilege.
++
+[role="screenshot"]
+image::images/ccs-local-role.png[Local cluster role configuration]
+
+.. *Remote cluster role*: Assign the `read` and `read_cross_cluster` privileges to the indices you want to search. You don't need to include the remote cluster's name here.
++
+[role="screenshot"]
+image::images/ccs-remote-role.png[Remote cluster role configuration]
+
+. On the local cluster:
+
+.. Assign the role you just created to a user who you want to configure your cross-cluster detection rules.
++
+[IMPORTANT]
+====
+* This step ensures that the privileges to read remote indices are applied from the user to the rule itself. When a user creates a new rule or saves edits to an existing rule, their current privileges are saved to the rule's API key. If that user’s privileges change in the future, you'll need to <<update-api-key,update the rule's API key>> to update the privileges that the rule uses.
+* This user must also have the <<enable-detections-ui,appropriate privileges>> to manage and preview rules.
+====
+
+.. With this user, <<rules-ui-create,configure a rule>> to search the remote indices. When creating or editing a rule, enter the `<remote_cluster_name>:<index_name>` pattern in the *Source* section.
++
+[role="screenshot"]
+image::images/ccs-rule-source.png[Rule source configuration]
++
+NOTE: If the rule's *Source* uses a data view instead of index patterns, you must also define the data view using the `<remote_cluster_name>:<index_name>` pattern. Refer to {kibana-ref}/data-views.html#management-cross-cluster-search[Use data views with cross-cluster search] for more on defining a data view.
```

Comment:
```suggestion
NOTE: If the rule's *Source* uses a data view instead of an index pattern, you still need to define the data view using the `<remote_cluster_name>:<index_name>` pattern. Refer to {kibana-ref}/data-views.html#management-cross-cluster-search[Use data views with cross-cluster search] for more on defining a data view.
```
I could be missing something here but I read this as meaning "If the rule's source uses a data view, use the same pattern you would have used had you not read this note." In which case I wouldn't be sure what the first sentence adds. To me it wasn't obvious after reading it why the user should go to the kibana docs.

Reply:
Good catch, I think I tried to cram too much into that first sentence. It's meant to say that data sources also require specific configuration to work with cross-cluster search, and that configuration is actually a separate process & UI from defining the rule. Let me take another pass at this...

Reply:
@benironside maybe something like this? Would it be better to omit "using the x:y pattern"? I wanted to keep reinforcing the sample pattern throughout, but we could omit if it's misleading or too much in one sentence.
```suggestion
NOTE: If the rule's *Source* uses a data view instead of index patterns, that data view must be defined separately, using the `<remote_cluster_name>:<index_name>` pattern. Refer to {kibana-ref}/data-views.html#management-cross-cluster-search[Use data views with cross-cluster search] for more on defining a data view.
```

Reply:
Do you think it'd be helpful lead-in to the Kibana doc link to specify that we're talking about a data view with cross-cluster search? For example:

```suggestion
NOTE: If the rule's *Source* uses a data view with cross-cluster search instead, you must define the data view separately, using the `<remote_cluster_name>:<index_name>` pattern. Refer to {kibana-ref}/data-views.html#management-cross-cluster-search[Use data views with cross-cluster search] for more on defining a data view.
```

Reply:
@nastasha-solomon I'd like to keep "uses a data view instead of index patterns" bc it directly references the UI options, but I think we could do something similar to what you suggested:
```suggestion
NOTE: If the rule's *Source* uses a data view instead of index patterns, you must define the data view for cross-cluster search separately, using the `<remote_cluster_name>:<index_name>` pattern. Refer to {kibana-ref}/data-views.html#management-cross-cluster-search[Use data views with cross-cluster search] for more on defining a data view.
```